src/Psy/Command/WhereamiCommand.php

```diff
@@ -108,8 +110,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $num = $input->getOption('num');
         $colors = new ConsoleColor();
         $colors->addTheme('line_number', array('blue'));
+        $colors->setForceStyle($this->forceColor());
         $highlighter = new Highlighter($colors);
         $contents = file_get_contents($info['file']);
         $output->page($highlighter->getCodeSnippet($contents, $info['line'], $num, $num), ShellOutput::OUTPUT_RAW);
     }
+
+    /**
+     * Sets whether or not to force colors in output.
+     *
+     * @param bool $forceColor
+     */
+    public function setForceColor($forceColor)
```

Comment:
I would personally use a constructor argument for this, rather than getters and setters. That simplifies things here as well as `Shell::getDefaultCommands` :)